package.json

```diff
@@ -50,14 +50,14 @@
     "src"
   ],
   "scripts": {
-    "compile": "rimraf out && tsc -p ./ && cp -R algosdk out/algosdk",
+    "compile": "shx rm -rf out && tsc -p ./ && shx cp -R algosdk out/algosdk",
     "lint": "eslint src --ext ts",
     "typecheck": "tsc -p tsconfig.json --noEmit",
     "check-format": "prettier . --check",
     "format": "prettier . --write",
     "extension:esbuild-base": "esbuild ./extension/src/extension.ts --bundle --tsconfig=./tsconfig.json --external:vscode --format=cjs --platform=node --outfile=extension/dist/extension.js",
+    "extension:esbuild-web": "esbuild ./extension/src/web-extension.ts --bundle --tsconfig=./tsconfig.json --external:vscode --format=cjs --platform=node --outfile=extension/dist/web-extension.js",
```

Comment:
Please revert this, this is meant to build for the browser

Reply:
Looks like it's the same as before but moved up in the file so it's next to the base build AFAICT

Reply:
I'm suprised this actually works in the browser, given the `AvmRuntime` extends `EventEmitter`.

Reply:
Reverted